Source/Core/Common/Timer.cpp

```diff
@@ -261,4 +261,13 @@ double Timer::GetDoubleTime()
 	return TmpTime;
 }
 
+std::string Timer::GetDateTimeFormatted(double time)
+{
+	time_t seconds = (time_t)time + (38 * 365 * 24 * 60 * 60);
+	struct tm * gmTime = localtime(&seconds);
```

Comment:
I get that most of this file already uses C time functions, but why not use std::chrono for new code?